python/tests/server/test_webhook.py

```diff
@@ -153,3 +156,28 @@ def test_webhook_caller_connection_errors():
     c = webhook_caller("https://example.com/webhook/123")
     # this should not raise an error
     c(response)
+
+
+@responses.activate
+def test_webhook_caller_converts_files_data_urls():
+    with tempfile.NamedTemporaryFile() as tmpfile:
+        with open(tmpfile.name, "w", encoding="utf8") as handle:
+            handle.write("hello world")
+
+        c = webhook_caller("https://example.com/webhook/123")
+
+        payload = {
+            "status": Status.SUCCEEDED,
+            "output": Path(tmpfile.name),
+            "input": {},
+        }
+        response = PredictionResponse(**payload)
+        payload["output"] = "data:None;base64,aGVsbG8gd29ybGQ="
```

Comment:
Is `None` supposed to be here?
```suggestion
        payload["output"] = "data:text/plain;base64,aGVsbG8gd29ybGQ="
```

Reply:
Actually what is this test doing?

Reply:
It tests that the webhook is sent data URLs rather than tmp file directories. I can change the mime-type however it seems odd to me that the `upload_file` function uses `None` as a default if that default is considered itself invalid. Can you comment on that?